src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/StringExtensionTests.cs

```diff
@@ -517,5 +517,64 @@ public void AsciiEncoding_Utf8Chars()
                 Assert.Equal("f7d6171b6bcfd8923a50b65721064b27", input.ToMd5Hash(Encoding.ASCII));
             }
         }
+
+        public class NaturalOrderTests
+        {
+            [Fact]
+            public void EmptyList()
+            {
+                string[] strings = [];
+
+                Assert.Equal([], strings);
+            }
+
+            [Fact]
+            public void OrdersSingleCharacters()
+            {
+                string[] strings = ["d", "1", "b", "c", "a"];
+                string[] expected = ["1", "a", "b", "c", "d"];
+
+                Assert.Equal(expected, strings.NaturalOrder());
+            }
+
+            [Fact]
+            public void OrdersMultipleCharacters()
+            {
+                string[] strings =
+                [
+                    "z24",
+                    "z2",
+                    "abcde",
+                    "z15",
+                    "abd",
+                    "z1",
+                    "z3",
+                    "abc",
+                    "z20",
+                    "z5",
+                    "abcd",
+                    "z11",
+                    "z22"
+                ];
+                string[] expected =
```

Comment:
Nice addition - I wonder if there are other places around the codebase that could use this.  Does the front end have any concept of natural ordering?